internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanger/scanners/qualys/App.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ *   Copyright (c) 2019, WSO2 Inc., WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ * /
+ */
+
+package org.wso2.security.tools.scanmanger.scanners.qualys;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
+ * Application for Qualys Micro Service.
+ */
+@SpringBootApplication
+public class App {
```

Comment:
Is this class required for Qualys scanner? If not can we remove? 

IMU, common module will act as the starting point of the scanner.